src/main/webapp/definitions/alfresco-model.yaml

```diff
@@ -0,0 +1,318 @@
+            },
+            "entries": [
+              {
+                "entry": {
+                  "name": "titled",
+                  "prefixedName": "cm:titled",
+                  "description": "Titled",
+                  "title": "Titled",
+                  "properties": [
+                    {
+                      "name": "title",
+                      "prefixedName": "cm:title",
+                      "title": "Title",
+                      "description": "Content Title",
+                      "dataType": "d:mltext",
+                      "facetable": "UNSET",
+                      "indexTokenisationMode": "BOTH",
+                      "multiValued": false,
+                      "mandatoryEnforced": false,
+                      "mandatory": false,
+                      "indexed": true
+                    },
+                    {
+                      "name": "description",
+                      "prefixedName": "cm:description",
+                      "title": "Description",
+                      "description": "Content Description",
+                      "dataType": "d:mltext",
+                      "facetable": "UNSET",
+                      "indexTokenisationMode": "BOTH",
+                      "multiValued": false,
+                      "mandatoryEnforced": false,
+                      "mandatory": false,
+                      "indexed": true
+                    }
+                  ]
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              },
+            ]
+          }
+        }
+        ```
+      operationId: listAspects
+      parameters:
+        - name: where
+          in: query
+          description: |
+            Optionally filter the list. Here are some examples:
+
+            The following where clause will only return aspects from the customModel1 and customModel2.
+              ```
+              where=(modelIds=customModel1,customModel2)
+              ```
+
+              The following where clause will only return all the aspect from custom models.
+              ```
+              where=(customModelOnly=true)
+              ```
+          required: false
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/AspectPaging'
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/aspect/{aspectId}':
+    get:
+      tags:
+        - aspects
+      summary: Get a aspect
+      description: |
+        **Note:** This is available in Alfresco 7.0.0 and newer versions.
+        Get the specific aspect **aspectId** on model.
```

Comment:
Please change to "Get information for aspect **aspectId**." i.e. consistent with /nodes/{nodeId}